selecta

```diff
@@ -276,6 +266,15 @@ class Search
     end
   end
 
+  def move(direction)
```

Comment:
Yeah, we probably should've stuck with +1/-1 here.

On Tue, Jan 20, 2015 at 8:10 AM, Markus Unterwaditzer <
notifications@github.com> wrote:

> In selecta
> <https://github.com/garybernhardt/selecta/pull/72#discussion-diff-23220588>
> :
>
> > @@ -276,6 +266,15 @@ class Search
> >      end
> >    end
> >
> > +  def move(direction)
>
>  Search#move is ambiguous
>
> IMO you could make the same argument for up/down.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/garybernhardt/selecta/pull/72/files#r23220588>.
>



-- 
Thanks,
Taylor Blau

ttaylorr.com